<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileVisualizer</class>
 <widget class="QWidget" name="FileVisualizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Large Data Visualizer V0.3      Â©Yanjie Ze </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Options:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Functions:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>560</y>
     <width>391</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>State Info</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="state_textEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>551</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>User ID from</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="user_id_lower"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>to</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="user_id_upper"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Location ID from</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="location_id_lower"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>to</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="location_id_upper"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Time from</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="time_lower"/>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="time_upper"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Latitude from</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="latitude_lower"/>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>to</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="latitude_upper"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Longitude from</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="longitude_lower"/>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>to</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="longitude_upper"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>291</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="top10Button">
      <property name="text">
       <string>Top 10 Popular POIs!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="checkinButton">
      <property name="text">
       <string>Checking-ins of Selected POIs!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearButton">
      <property name="text">
       <string>Clear Cavans</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>661</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>791</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="drawingLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
